.coveragerc

```diff
@@ -17,7 +17,6 @@ omit=
     # We can't unit test our uis, so we're not going to put them inside the coverage
     *tank/authentication/ui/*
     *tank/authentication/sso_saml2/*
-    *tank/platform/qt/*
```

Comment:
I'm not sure I understand why we want to stop omitting this directory?

Reply:
Back in the day we believed we wouldn't/cloudn't unit test our GUIs. Well, here we are. 😄

Reply:
👍 A little slow on the uptake today...